stable/drupal/templates/drupal-pvc.yaml

```diff
@@ -25,4 +17,13 @@ spec:
   resources:
     requests:
       storage: {{ .Values.persistence.drupal.size | quote }}
+{{- if (not .Values.persistence.drupal.hostPath) }}
+{{- if .Values.persistence.drupal.storageClass }}
```

Comment:
could be
```
{{- if and .Values.persistence.drupal.storageClass (not .Values.persistence.drupal.hostPath) -}}
```
you might need to wrap the `and` in parens too, be sure to test it

Reply:
Done, thanks @prydonius